docs/mint.json

```diff
@@ -70,6 +70,10 @@
     {
       "name": "All Tools",
       "url": "https://composio.dev/tools"
+    },
+    {
+      "name": "Chat with Code",
+      "url": "https://chatgpt.com/g/g-67697db23c808191a1787ea4b86ac1ce-composio"
```

Comment:
Consider using a more permanent or branded URL for the ChatGPT integration. The current raw GPT instance URL might be subject to change and could break in the future. It would be better to either:
1. Use a redirect URL through your domain (e.g., chatgpt.composio.dev)
2. Add a comment explaining the purpose of this specific GPT instance
3. Consider versioning this integration in case the GPT model gets updated